os/kernel/init/os_bringup.c

```diff
@@ -276,7 +276,13 @@ static inline void os_do_appstart(void)
 #ifdef CONFIG_SECURITY_LEVEL
 	int ret = set_security_level();
 	if (ret != SECURITY_LEVEL_OK) {
-		dbg("[set_security_level] error ret : %d\n", ret);
+		if (ret == SECURITY_LEVEL_READ_FAILED) {
```

Comment:
set_security_level() This function has a total of 4 return values.
1 success value, 3 failure values So, I think, why not print an error message for else as well? (ALLOC_FAILED)

I only offer opinions, you don't have to accept them.

Reply:
I added log for alloc failed. Thank you for your review.